src/unofficial.md

```diff
@@ -37,6 +37,8 @@ Introductory:
 * [The Node Experiment: Exploring Async Basics with Rust](https://web.archive.org/web/20230125023131/https://cfsamson.github.io/book-exploring-async-basics/) - from the internet archive
 * [Rust Exercises by Ferrous Systems](https://rust-exercises.ferrous-systems.com/)
 * [Rust Training Slides by Ferrous Systems](https://rust-training.ferrous-systems.com/latest/slides/)
+* [Rust Atomics and Locks](https://marabos.nl/atomics/) - Low-Level Concurrency in Practice
+* [Rust for Rustaceans](https://rust-for-rustaceans.com/)
```

Comment:
The linked page is a review of a hardcopy book, which belongs in https://github.com/sger/RustBooks, rather than here. Indeed, it's already there.